src/OpenApi/Model/PathItem.php

```diff
@@ -184,7 +184,7 @@ public function withServers(?array $servers = null): self
         return $clone;
     }
 
-    public function withParameters(array $parameters): self
+    public function withParameters(?array $parameters = null): self
```

Comment:
@soyuka you need to change the return type of `getParameters()` to include `null`

Reply:
damn I wish I could have phpstan level7 thanks!

Reply:
Add it! Baseline everything. Fix old code slowly

Reply:
there's a baseline generator? I think we're close to level 7 anyways, it's just the "document array type" rule that is missing in a lot of files.

Reply:
`--generate-baseline` flag!